project/MultiNode.scala

```diff
@@ -73,6 +73,14 @@ object MultiNode extends AutoPlugin {
         testResults.events ++ multiNodeResults.events,
         testResults.summaries ++ multiNodeResults.summaries)
     })
+
+  implicit class TestResultOps(val self: TestResult) extends AnyVal {
+    def id: Int = self match {
+      case TestResult.Passed ⇒ 0
+      case TestResult.Failed ⇒ 1
+      case TestResult.Error  ⇒ 2
```

Comment:
Great investigative work! I will borrow this for the akka build conversion to sbt 1.x. :)